wiki/Contests/Monthly_Beatmapping_Contest/en.md

```diff
@@ -0,0 +1,79 @@
+
+Since 2020, the MBC has adopted a new schedule, ruleset, voting procedure, and reward scheme. The rest of this article details the current format for Monthly Beatmapping Contests.
+
+## Schedule
+
+The timeline for each month's contest is as follows:
+
+- **Beatmapping (3 weeks):** Contest is announced at the start of the month and beatmappers create their entries.
+- **Screening (1 week):** Entries are evaluated by experienced beatmappers.
+- **Voting (2 weeks):** Top entries from screening are published for the osu! community to vote on.
+- **Rewards:** Winners are announced and prizes are redeemed.
+
+The voting portion of each contest overlaps with the next month's contest's beatmapping portion.
+
+## Ruleset
+
+Submissions for the MBC must abide by the following rules:
+
+- **Submissions must follow the month's beatmapping limitation.** The limitation differs month by month and is detailed on each contest's announcement post.
+- **Submissions must be a single difficulty from one of the month's selected songs.** Selected songs differ month by month and are included in each contest's announcement post.
+- **Submissions must be in `.osz` format.** Storyboarded content will not be considered.
+- **Submissions must be in the osu! game mode.** Other game modes will be considered after trialing this contest with osu! only.
+- **Submissions must *not* be uploaded to the osu! website through the Beatmap Submission System (*BSS*).** Creators of each entry are anonymous during the screening and voting processes.
+- **Submissions must abide by the [ranking criteria](/wiki/Ranking_Criteria).**
+
+One or more rules may be added or dropped depending on the month's specifications. Any exceptions will be clearly explained in the contest's announcement post.
+
+## Results
+
+Results for the MBC are first narrowed down to the most competitive entries, then published for community vote. Creators of all entries are anonymous in the screening and voting stages.
+
+### Screening
+
+A group of at least 10 experienced beatmappers (*screening crew*) are tasked with reviewing all entries and providing their top picks in order of preference. Entries that receive the most praise from the screening crew will move on to the voting stage.
+
+### Voting
+
+The amount of beatmaps that advance to the voting stage varies depending on the amount of highly competitive submissions the contest receives and will usually range between 5 and 10 finalists.
+
+Voting takes place on the respective contest's [Community Contests page](https://osu.ppy.sh/community/contests). Users are encouraged to vote for their favorite entry and to look out for any entries that effectively work with the month's beatmapping limitation.
+
+### Previous contests
```

Comment:
feels weird that this isn't its own level 2 header

Reply:
moved it, hopefully better